packages/build/tests/edge_functions/tests.js

```diff
@@ -136,3 +137,17 @@ test.serial('writes manifest contents to system logs if `systemLogFile` is set',
     /Edge Functions manifest: {"bundles":\[{"asset":"[a-fA-F\d]{64}\.js","format":"js"}],"routes":\[{"function":"function-1","pattern":"\^\/one\/\?\$"}],"bundler_version":"\d+\.\d+\.\d+"}/,
   )
 })
+
+test('build plugins can manipulate netlifyToml.edge_functions array', async (t) => {
+  const fixtureName = 'functions_plugin_mutations'
+
+  await runFixture(t, fixtureName, {
+    flags: { debug: false },
```

Comment:
Rather than manually verifying the manifest contents, you could set `debug: true` here, which will print the manifest to stdout. You could then use the snapshot comparison and remove most of the logic in the test.

It's totally fine to keep it as is though, just thought I'd mention it.

Reply:
Yeah I thought about doing that at first, but I think this is a bit more explicit to the user about what is going on :)